### PR TITLE
in case when payload[key] == null error exists

### PR DESCRIPTION
**Pull request by logovazzik**

### CODE REVIEW

**Comment by gdouezangrard** [comment]
linking #305

File: core/network/client.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ function includesMedia (payload) {
   return Object.keys(payload).some(
     (key) => Array.isArray(payload[key])
       ? payload[key].some(({ media }) => typeof media === 'object' && (media.source || media.url))
-      : typeof payload[key] === 'object' && (payload[key].source || payload[key].url)
+      : typeof payload[key] === 'object' && payload[key] && (payload[key].source || payload[key].url)
```
Comment (Piterden) [inline core/network/client.js]: `payload[key] && typeof payload[key] === 'object' && (payload[key].source || payload[key].url)`

Comment (logovazzik) [reply core/network/client.js]: changed

Comment (Piterden) [reply core/network/client.js]: Even
```js
!!payload[key] && typeof payload[key] === 'object' && (payload[key].source || payload[key].url)
```

Comment (Piterden) [reply core/network/client.js]: No. Damm!

**Pull request closed without merging**